crates/executor/src/execution_state.rs

```diff
@@ -46,6 +46,9 @@ mod versioned_constants {
     const BLOCKIFIER_VERSIONED_CONSTANTS_JSON_0_13_3: &[u8] =
         include_bytes!("../resources/versioned_constants_0_13_3.json");
 
+    const BLOCKIFIER_VERSIONED_CONSTANTS_JSON_0_13_4: &[u8] =
+        include_bytes!("../resources/versioned_constants_0_13_4.json");
+
```

Comment:
Hmm, why do we need this? Seems redundant with `VersionedConstants::latest_constants()` below and seems to break `custom_versioned_constants`.